### ISSUE TITLE
Using %a in statusline to show the free space gets a wrong value

### ISSUE THREAD

**Issue by yanzhang0219** [vifm/vifm#546]
Hi,

Based on the doc [here](https://vifm.info/vimdoc.shtml#vifm-%27statusline%27), I want to use `%a` in the statusline to show the amount of free space, but I get a weird number `72 T` there. My SSD is 512 so this 72 thing is weird. 

My config about statusline:
![image](https://user-images.githubusercontent.com/11582667/80857826-df780300-8c09-11ea-9076-afab35d28e71.png)

The statusline result:
![image](https://user-images.githubusercontent.com/11582667/80857813-ce2ef680-8c09-11ea-9e84-e91b5e82bb74.png)

Thank you~

**Comment by xaizek** [comment]
Hello,

thanks for the bug report. I didn't realize that on OS X (which you mentioned in another issue) API for querying file-system information has different meaning of fields compared to other systems, which is why you get wrong results.

**Comment by yanzhang0219** [comment]
Oh yeah, sorry I forgot to put my OS version there. Thank you.

**Comment by yanzhang0219** [comment]
Could you please fix it on the master so that I can rebuild it? Thanks!!

**Comment by yanzhang0219** [comment]
Hi,

Thank you for fixing it. And one more issue comes up...

After I run git pull the latest commit from the master and rebuilt it, and when I tried to open vifm, an error popped up:
![image](https://user-images.githubusercontent.com/11582667/80998777-625eb080-8df8-11ea-9512-f1d9f9ee192b.png)

The icons file is [here](https://github.com/yanzhang0219/dotfiles/blob/master/.config/vifm/icons). And if in the icons file I appended an empty line at the end, this error was fixed. 
![image](https://user-images.githubusercontent.com/11582667/80999037-d39e6380-8df8-11ea-8e9d-495c37ceb29c.png)
